lib/src/zenohRpcClient.cpp

```diff
@@ -148,9 +152,20 @@ std::future<RpcResponse> ZenohRpcClient::invokeMethodInternal(const UUri &topic,
 
     auto uuid = Uuidv8Factory::create();
 
-    auto builder = UAttributesBuilder::request(topic /* TODO change to the entity */, topic, options.priority(), options.ttl());
+    auto client_uri = BuildUUri()
+                .setAutority(clientAuthority)
+                .setEntity(BuildUEntity()
+                        .setName("test_rpc.client")
+                        .setMajorVersion(1)
+                        .setId(1)
+                        .build())
+                .setResource(BuildUResource()
+                        .setRpcRequest("handler", 1)
+                        .build())
+                .build();
```

Comment:
This looks like test code in the production code. What is this doing here?

Reply:
handler is wrong. Response needs to be .setRpcResponse() which will populate the name, instance and ID correctly (if we are talking about the RPC response).

Reply:
I've changed to setRpcResponse(), but it doesn't compile.  When I look at the BuildResource.h, it doesn't seem to match expectations. Steven Hartley, can you clarify about this?

Reply:
> I've changed to setRpcResponse(), but it doesn't compile. When I look at the BuildResource.h, it doesn't seem to match expectations. Steven Hartley, can you clarify about this?

what do you mean? the API seems to be right....